src/utils/combineClasses.ts

```diff
@@ -0,0 +1 @@
+export const combineClasses = (classArray: string[]): string => classArray.join(' ');
```

Comment:
If you change it to this
`export const combineClasses = (...classArray: string[]): string => classArray.join(' ');
`
you will be able to pass parameters without array
`combineClasses('class1', 'class2', 'class3');`

Reply:
thanks, nice suggestion)
fixed